cognite/powerops/custom_modules/power_model_v1/data_models/views/function_inputs/BenchmarkingShopRunInputDayAhead.view.yaml

```diff
@@ -0,0 +1,65 @@
+space: "{{power_ops_models}}"
+externalId: BenchmarkingShopRunInputDayAhead
+name: BenchmarkingShopRunInputDayAhead
+description: Class for inputs to shop runs used for day ahead benchmarking
+filter:
+  and:
+    - hasData:
+        - type: container
+          space: "{{power_ops_models}}"
+          externalId: FunctionData
+    - equals:
+        property:
+          - node
+          - space
+        value: "{{power_ops_instance_space}}"
+    - equals:
+        property:
+          - node
+          - type
+        value:
+          externalId: BenchmarkingShopRunInputDayAhead
+          space: "{{power_ops_type_space}}"
+implements:
+  - space: "{{power_ops_models}}"
+    externalId: ShopTriggerInput
+    version: "{{version}}"
+    type: view
```

Comment:
**issue:** if this is going to implement `ShopTriggerInput` then the view file name for `ShopTriggerInput` needs to be updated to `2-interface.ShopTriggerInput.view.yaml` to ensure that the resources will be created in sequential order to be able to reference it.

Reply:
**question:** _why_ is this view implementing from `ShopTriggerInput`? what fields from `ShopTriggerInput` does it actually need and use?

Reply:
because the common function `create_cogshop_tasks_definition_dm` expects a list of ShopTriggerInput instances, which are sent to the function `fdm_shop_trigger`, and `fdm_shop_trigger` fetches shop trigger instances. However, the extra fields in `BenchmarkingShopRunInputDayAhead` are needed for further processing within benchmarking.

Reply:
Then my question is why is that further processing happening within the context of the `ShopTrigger` function?